contracts/core/liquidators/impl/Auction.sol

```diff
@@ -205,7 +242,7 @@ contract Auction {
         uint256 currentSetNaturalUnit = _currentSet.naturalUnit();
         uint256 nextNaturalUnit = _nextSet.naturalUnit();
         return Math.max(currentSetNaturalUnit, nextNaturalUnit)
-            .mul(pricePrecision);
+            .mul(_pricePrecision);
     }
 
     /**
```

Comment:
Do we still need the definition of price to be based on a numerator and denominator if we're removing the ExponentialPivot?

Reply:
Yes decaying the denominator was just a way to make it go exponential. But it's still relevant because a numerator of 450 could mean a price equal to .45 (with denominator 1,000) or price of .045 (with a denominator of 1,0000)